Formula/mpd.rb

```diff
@@ -92,6 +97,11 @@ def install
     args << "--enable-nfs" if build.with? "libnfs"
     args << "--enable-modplug" if build.with? "libmodplug"
     args << "--enable-pulse" if build.with? "pulseaudio"
+    if build.with? "upnp"
+      args << "--enable-upnp"
+      args << "--enable-expat"
+      args << "--with-expat=/usr/local/opt/expat"
```

Comment:
Given that there is a `depends_on "expat"` above, the configure script should find it without help. Please try if it works that way.

If it does not, we'll keep the `--with-expat` but please use `--with-expat=#{Formula["openssl"].opt_prefix}` rather than hardcoding the path.